src/component/handlers/edit/editOnPaste.js

```diff
@@ -102,7 +102,7 @@ function editOnPaste(editor: DraftEditor, e: DOMEvent): void {
   if (text && !html) {
     // The pasted content has text, but not HTML. For certain browsers (old versions of Safari, IE, and Edge)
     // the html isn't provided as part of the clipboardData. To work around this, follow the following algorithm:
-    // Do NOT call e.preventDefault(). Instead, we want the browser to paste, just not in the draft element.
+    // Do NOT call e.preventDefault(). Instead, we want the browser to paste, just not in the editor element.
     // Instead, move focus to a dummy contentEditable div (the pasteTrap),
     // let the paste event through, and then copy the html out of the paste trap.
     // It is important to call setMode('paste') to disable draft's event handlers (so it is blisfully unaware)
```

Comment:
Minor: Same comment as earlier. We shouldn't talk about Draft as a 3rd party here.